.github/workflows/database.yml

```diff
@@ -0,0 +1,42 @@
+name: Database CI
+on: push
```

Comment:
The file contains a key `on: ??` which defines on which [events](https://docs.github.com/en/actions/reference/events-that-trigger-workflows) should this action occur.

Common ones are `on: push` and `on: pull_request`, but there are more than that (e.g. `on: schedule` is useful for, say, generating a "nightly" release every night).

Some events take more options, but for our purposes, we just want every commit that is pushed to be checked, so we do `on: push`. It matters more to only run some workflows on more specific events when it's a private repository, in which case Github charges you per second spent with actions running (yikes).